kotlin/workflow-testing/src/main/java/com/squareup/workflow/testing/MockWorker.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2019 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.workflow.testing
+
+import com.squareup.workflow.Worker
+import com.squareup.workflow.Worker.Emitter
+
+/**
+ * A mock implementation of [Worker] for use in tests with `testRender` and [TestRenderResult].
+ *
+ * Not this [Worker] can not actually emit any output itself. Use
```

Comment:
Typo on `Not` -> `Note` (I believe this should read "Note this [Worker] can not actually emit any output...")

Reply:
Fixed.